packages/botonic-react/tests/webchat/usewebchat.test.js

```diff
@@ -167,6 +167,22 @@ describe('TEST: useWebchat ', () => {
     expect(result.current.webchatState.isWebchatOpen).toEqual(true)
   })
 
+  it('toggleEmojiPicker: assign true to webchatState.isEmojiPickerOpen', () => {
+    const { result } = renderUseWebchatHook()
+    act(() => {
+      result.current.toggleEmojiPicker(true)
+    })
+    expect(result.current.webchatState.isEmojiPickerOpen).toEqual(true)
+  })
+
+  it('togglePersistentMenu: assign true to webchatState.isPersistentMenuOpen', () => {
+    const { result } = renderUseWebchatHook()
+    act(() => {
+      result.current.togglePersistentMenu(true)
+    })
+    expect(result.current.webchatState.isPersistentMenuOpen).toEqual(true)
```

Comment:
I would add an initial 
    expect(result.current.webchatState.isPersistentMenuOpen).toEqual(false)
 so that it's clear that toggle changes the previous state